index.rst

```diff
@@ -1,14 +1,14 @@
-:tocdepth: 1
+######################################################
+Velocity, Acceleration, and Jerk analysis of TMA slews
+######################################################
 
-.. sectnum::
+.. abstract::
 
-.. Metadata such as the title, authors, and description are set in metadata.yaml
+   This technote shows analysis of velocity, acceleration, and jerk of the TMA during random walk slews of 3.5 degrees on multiple nights.  Most slews are within the specifications, but some fail, probably because of noisy values in the encoders.
 
 
-Abstract
-========
 
-This technote shows analysis of velocity, acceleration, and jerk of the TMA during random walk slews of 3.5 degrees on multiple nights.  All slews are within the specifications.
+.. Metadata such as the title, authors, and description are set in metadata.yaml
```

Comment:
line 18: update the .rst  with   "The analysis and the plots presented in this technote are available in the notebook  `SITCOMTN-067_Vel_Accel_Jerk_15Nov23.ipynb <https://github.com/lsst-sitcom/notebooks_vandv/tree/develop/notebooks/tel_and_site/subsys_req_ver/tma>`_ " 


Basically include a hyperlink to the entire path. I have also corrected the full path. N

Reply:
line 28: italicize "scipy.interpolate.UnivariateSpline.derivative"